examples/personalization_fedloadforecast.py

```diff
@@ -0,0 +1,297 @@
+import os
+import time
+import argparse
+import numpy as np
+import torch
+from mpi4py import MPI
+import copy
+import io
+
+from appfl.config import *
+from appfl.misc.data import *
+from appfl.misc.utils import load_model_state
+import appfl.run_serial as rs
+import appfl.run_mpi as rm
+
+from models.cnn import *
+from models.utils import get_model
+from models.utils import validate_parameter_names
+
+from losses.utils import get_loss
+from metric.utils import get_metric
+
+from datasets.PreprocessedData.NREL_Preprocess import NRELDataDownloader
```

Comment:
Could you please also include code for NRELDataDownloader

Reply:
Ah I just saw this. Looks like PreprocessedData is blacklisted in .gitignore. will add